lib/linter.js

```diff
@@ -63,7 +58,9 @@ class Linter {
 
         const tree = this.parse(source);
 
-        this.ast = tree.root.toResult();
+        this.ast = tree.root.toResult({
+            stringifier: stringify
```

Comment:
I'm not sure either. Normal stringifying using postcss-less is done as seen here: https://github.com/shellscape/postcss-less#stringifying. Does it look correct using that method? Looking at the docs for `Result` (http://api.postcss.org/Result.html) it looks like it's a representation of the css, so specifying a stringifier seems logical.